### ISSUE TITLE
500 returned when querying outputs for a custom container

### ISSUE THREAD

**Issue by georgesnelling** [lyft/flyte#49]
There is an expectation from Admin that _some_ type of output will exist in storage for a NodeExecution. This turns out not to be the case if a container is running without the SDK. We need some type of handling for this case.

**Comment by kumare3** [comment]
I do not think this is valid or true any more. Should we resolve this? @matthewphsmith
